src/components/Editor/index.js

```diff
@@ -188,11 +197,14 @@ class Editor extends Component {
   }
 
   render() {
-    const { contentEditable, className, style } = this.props
+    const { contentEditable, className, style, ...rest } = this.props
     const { html } = this.state
+    delete rest.children;
```

Comment:
This is a rather non obvious v8 perf hog :) can you just manually set code to undefined for presentations below? Also you don't need to do anything about children